js/web/lib/wasm/jsep/webgpu/ops/fuse-utils.ts

```diff
@@ -38,7 +41,24 @@ export const parseInternalActivationAttributes =
 
       if (activation === 'Clip') {
         const [clipMin, clipMax] = attributes?.activation_params as [number, number] || [MIN_CLIP, MAX_CLIP];
-        return {activation, clipMax, clipMin, activationCacheKey: `${activation}:${clipMin},${clipMax}`};
+        return {activation, clipMax, clipMin};
+      }
+      return {activation};
+    };
+
+export const updateUniformsFromActivation =
+    (programUniforms: ProgramUniform[], uniforms: UniformsArrayType, attributes: InternalActivationAttributes,
+     dataType: number) => {
```

Comment:
From the spec of [com.microsoft.FusedConv](https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md#commicrosoftfusedconv), `activation_params : list of floats`, so you can directly use float32 as the type.
In your current implementation, the dataType may be a float16, which is not supported in current uniform, which will bring regressions since we support float16 before.

Another concern is that uniforms in shader won't change if the shader key doesn't change. However, programUniforms need to be updated every time. It's not that good to update them together every time.

Reply:
Turn activation_params to float is done. 
Will handle uniforms/programUniforms in a split PR.